### PR TITLE
226209 pr

### PR DESCRIPTION
**Pull request by GodsonLeigh**
Changed content to reflect new delinked payment reduction calculator
Added new Delinked payments button
Edited further information link to new content page

### CODE REVIEW

File: test/unit/home-content.test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+const homeContent = require('../mocks/mockHomeContent')
```
Comment (AbidemiAdio) [inline test/unit/home-content.test.js]: Your reference is to mockHomeContent, hence, you are actually testing mockHomeContent but not homeContent. The intention is to test homeContent.

Comment (GodsonLeigh) [reply test/unit/home-content.test.js]: I have tweaked the test file to include checks for the rendered page as discussed

File: test/mocks/mockHomeContent.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    toReceiveDelinked: 'To receive delinked payments, you must also have claimed, and been eligible for, BPS 2023 in England (except for some inheritance cases).',
+    linkToGuidance: 'Read [link to guidance] for more information about delinked payments.',
+    requireHelp: 'If you require help in using this calculator, please call the Rural Payments helpline on 03000 200 301'
+  },
+  opportunities: {
+    ifFarmer: 'If you\'re a farmer, land manager or forester, read Funding for farmers and land managers for information about opportunities to apply for money to improve the environment and your farms productivity. You can keep up to date with scheme news on the Rural Payments Agency website and the Defra Farming blog.',
+    moreInfo: 'If you would like more information about schemes and grants funded through the reductions to BPS, please visit the Rural payments and grants page'
+  }
+}
```
Comment (AbidemiAdio) [inline test/mocks/mockHomeContent.js]: None of the content of this mock file is referenced anywhere, could you please remove?

Comment (GodsonLeigh) [reply test/mocks/mockHomeContent.js]: Mock file removed

**Review by AbidemiAdio — DISMISSED** [review]
QED

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**